contents/blog/validate-b2b-product-market-fit.md

```diff
@@ -0,0 +1,74 @@
+
+**Does your product generate significantly more value for the audience than it costs?**
+
+This question revolves around value and cost, but how do we measure these? Look for customers giving the following indicators:
+- **Paying:** They’re willing to pay in full. If you’re retaining >80% of your customers month-on-month, that's a strong indicator that your product is generating more value than cost.
+- **References:** They’re willing to put their reputation on the line to recommend your product to their peers.
+- **Super-engaged:** They love your product, give you regular feedback, and tell the world how great it is without prompting.
+
+If your focus segment has a large number of small clients, you’ll want survey responses or data from at least 50 unbiased customers. There are a few options for common surveys used to validate product market fit:
+- **Net Promoter Survey(NPS):** If you ask your customers whether they would recommend your product, you want a higher ratio of people recommending than not-recommending.
+- **Disappointment:** If you ask your customers how disappointed they would be if they couldn’t use your product any more, you want >40% to suggest they would be very disappointed.
+- **Retention:** Are your customers continuing to use your product month over month? Retaining over 60% of users of a free version of your product is generally a strong indicator of value generation.
+
+## Pitfalls to avoid
+The following pitfalls can lull you into thinking you have product market fit when you don’t:
+- **Product customization:** It’s essential to validate your “vanilla” product. Any customizations or bespoke work you do to please individual customers will dent confidence in whether you have product market fit or just a great relationship with your customers.
+- **Wrong market:** Great fit within a highly niche market or one with limited spending power may hamper your company’s growth.
+- **Discounts:** It can be tempting to discount your product for your first few customers, but this harms any product validation tests. Good-fit customers should be willing to pay full price for your product - although getting pricing right can be hard.
+- **Novelty effects:** People using a new product or seeing a new feature for the first time might react differently than people who've been using the product for a long time. It’s important to give users time to get used to these new features (and perhaps stop deriving value from them) before calling product market fit.
+- **Friends and family:** Although these two groups can be a great way to kick-start adoption and get early feedback on your product, it’s important to determine whether their success with the product is due to knowing you or because the product actually solves their needs. Ensure your product passes the [Mom Test](http://momtestbook.com/ before calling product-market fit.
```

Comment:
This is a good book! Link formatting needs slight adjustment, though: missing `)`

Reply:
Fixed, good catch!